### ISSUE TITLE
Error in table 8 of the paper with the 300W NME?

### ISSUE THREAD

**Issue by jmbuena** [zhiqiangdon/CU-Net#7]
Hi,

I have looked into table 8 in your paper and all the measures you put there from the literature computes the NME dividing by the interpupil distance. On the other hand, I'm not sure if instead you are dividing by the outer eyes corners distance to compute your own results on table 8. In fact I'm not sure if the stacked hourglass results of reference [23] in your paper has the same normalization (dividing by outer eyes corners distance). Take into account that this is not comparable (see our ECCV 2018 [paper](http://openaccess.thecvf.com/content_ECCV_2018/papers/Roberto_Valle_A_Deeply-initialized_Coarse-to-fine_ECCV_2018_paper.pdf) as both results differ in more than 1% for the same method.

I ask it because at function [per_image_rmse](https://github.com/zhiqiangdon/CU-Net/blob/2ee2f9a206dcb8bad7c1dfcbb6f1b2ed086c2fae/pylib/FaceAcc.py#L64), you divide the error by the distance between landmarks 37 and 46 (36 y 45 in python indexes). And in [300W](https://ibug.doc.ic.ac.uk/media/uploads/images/annotpics/figure_68_markup.jpg) those are the outer eye corners landmarks.

Am I missing something?

Thanks a lot for any clarification about it.

**Comment by Sierkinhane** [comment]
@jmbuena I agree with you. I think the author took the inter-ocular results to compare with other's inter-pupil results.
